### PR TITLE
Mark fields as read-only

### PR DESCRIPTION
**Pull request by davkean**
Used the new code fix.

### CODE REVIEW

File: src/Microsoft.VisualStudio.Editors/XmlIntellisense/XmlIntellisense.vb [hunk 1]
```diff
@@ -105,7 +105,7 @@ Namespace Microsoft.VisualStudio.Editors.XmlIntellisense
     '   Methods on this class should only be called on the VS foreground
     '   thread.
     '--------------------------------------------------------------------------
-    <ClassInterface(ClassInterfaceType.None)> _
+    <ClassInterface(ClassInterfaceType.None)>
```
Comment (Pilchie) [inline src/Microsoft.VisualStudio.Editors/XmlIntellisense/XmlIntellisense.vb]: It's interesting that this seems to be pretty-listing the whole file.  @jinujoseph potential bug for you here.

**Comment by Pilchie** [comment]
Oops - this got merged into d15.7.x by mistake :(

@jasonmalinowski @dotnet/roslyn-infrastructure what should we do?

**Comment by davkean** [comment]
Why is that mistake?

**Comment by Pilchie** [comment]
Because 15.7 is in division escrow now, and this isn't the sort of thing that would meet that bar.

**Pull request merged**